### ISSUE TITLE
prepare stage broken on master

### ISSUE THREAD

**Issue by snicoll** [spring-projects/spring-boot#6055]
If you run `mvn -Pprepare install` on master, the `build.gradle` of `spring-boot-sample-actuator` is broken. This has been introduced by https://github.com/spring-projects/spring-boot/commit/58ca9a1c5df4d58492cf6f33ed60997afd1aa32e#diff-5ef98fe29c1b7f3fb347fe34d4277237R28

``` diff
diff --git a/spring-boot-samples/spring-boot-sample-actuator/build.gradle b/spring-boot-samples/spring-boot-sample-actuator/build.gradle
index 6413d1c..bbf5574 100644
--- a/spring-boot-samples/spring-boot-sample-actuator/build.gradle
+++ b/spring-boot-samples/spring-boot-sample-actuator/build.gradle
@@ -25,9 +25,7 @@ jar {
 }

 group = 'org.springframework.boot'
-version = springBootVersion
-
-repositories {
+version = springBootVersion = '1.4.0.BUILD-SNAPSHOT'
        // NOTE: You should declare only repositories that you need here
        mavenLocal()
        mavenCentral()
```

**Comment by wilkinsona** [comment]
`spring-boot-sample-actuator` also fails to build using Gradle as `SampleActuatorApplicationTests.testInfo` checks for things that are only contributed when built with Maven. I think we either need to fix the various build.gradle scripts and add tests that make sure they work or we should remove them.

**Comment by snicoll** [comment]
I like the fact that build-specific features are illustrated in the samples so in this particular case I'd like we keep it if we can.

**Comment by snicoll** [comment]
This was fixed as part of releasing 1.4.0
